app/controllers/application_controller.rb

```diff
@@ -2,9 +2,126 @@
 
 class ApplicationController < Sinatra::Base
 
-  configure do
-    set :public_folder, 'public'
-    set :views, 'app/views'
-  end
+	configure do
+		set :public_folder, 'public'
+		set :views, 'app/views'
+		enable :sessions
+	end
 
-end
\ No newline at end of file
+	get '/' do
+		erb :index
+	end
+
+	get '/signup' do
+		if !session[:id].nil?
```

Comment:
I don't see a `current_user` or `logged_in?` helper method, but they can be helpful when dealing with authenticated users. A bit more semantic than looking up session variables.